pandas/tests/extension/decimal/array.py

```diff
@@ -108,6 +108,8 @@ def astype(self, dtype, copy=True):
 
     def __setitem__(self, key, value):
         if pd.api.types.is_list_like(value):
```

Comment:
should prob import these like we normally do (is_like_like) at the top, but no big deal